cd/deploy.sh

```diff
@@ -4,29 +4,27 @@ set -e
 
 # Gets project root directory by calling two nested "dirname" commands on the this file 
 getRootDirectory() {
-    echo "$( cd "$(dirname "$(dirname "${BASH_SOURCE[0]}")")" > /dev/null 2>&1 && pwd )"
+    echo "$( cd $(dirname $(dirname $0)) > /dev/null 2>&1 && pwd )"
 }
 
 # Pushes Tags to Dockerhub
 pushTags() {
     name=$1
     version=$2
     app_version=$3
-
-    tags="$(echo "${version}" | sed -e "s/\(.*\)\.\(.*\)\..*/"${name}":"${version}"+"${app_version}" "${name}":"${version}" "${name}":\1.\2 "${name}":\1 "${name}":latest/")"
-
+    tags="$(echo ${version} | sed -e "s|\(.*\)\.\(.*\)\..*|${name}:${version}+${app_version} ${name}:${version} ${name}:\1.\2 ${name}:\1 ${name}:latest|")"
     docker tag "${name}" "${tags}"
```

Comment:
I'm getting the error: `Error parsing reference: "gchq/hdfs:3.2.1+0.5.0 gchq/hdfs:3.2.1 gchq/hdfs:3.2 gchq/hdfs:3 gchq/hdfs:latest" is not a valid repository/tag: invalid reference format`. I don't think you can create multiple tags at once. Might have to loop through them, tagging and pushing each individually.